pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -995,3 +998,28 @@ func (b *SlirpPodInterface) setCachedVIF(pid, name string) error {
 func (s *SlirpPodInterface) setCachedInterface(pid, name string) error {
 	return nil
 }
+
+func purgeUserDataIPv6Config(vmi *v1.VirtualMachineInstance) {
+	volumes := vmi.Spec.Volumes
+
+	for _, volume := range volumes {
+		if volume.VolumeSource.CloudInitNoCloud != nil {
+			userData := volume.VolumeSource.CloudInitNoCloud.UserData
+
+			if strings.Contains(userData, "ip -6 addr add") {
+				re := regexp.MustCompile(`sudo ip -6 addr add .* dev eth0`)
```

Comment:
First line doesnt have sudo in cloudinit so it wont match ?
+               ipv6UserDataString = fmt.Sprintf(`ip -6 addr add %s dev eth0
+                             sudo ip -6 route add default via %s src %s